recipe/meta.yaml

```
@@ -1,42 +1,59 @@
-{% set version = "23.2.0" %}
-{% set sha256 = "317b881c6715ca78222efe191310cacb3b1c463576c5bc70a14412a7dca6aaee" %}
+{% set version = "19.2.0" %}
+{% set sha256 = "736acf9cd273ad7ac6b34a9e2e8e0648eed7c34488d979683d192654fd50b2c2" %}
 
 package:
   name: structlog
   version: {{ version }}
 
 source:
-  fn: structlog-{{ version }}.tar.gz
-  url: https://github.com/hynek/structlog/archive/{{ version }}.zip
+  url: https://github.com/hynek/structlog/archive/{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
+  skip: True  # [py<37]
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir --no-build-isolation -v
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
     - hatchling
     - hatch-vcs
-    - hatch-fancy-pypi-readme
 
   run:
-    - python >=3.7
+    - python
 
 test:
+  requires:
+    - pip
+    - pytest
+    - freezegun >=0.2.8
+    - pretend
+    - twisted
```

```suggestion
    - twisted
    - python-rapidjson  # [py>=36]
```
https://github.com/hynek/structlog/blob/19.2.0/setup.py#L42C10-L42C26